Source/Mosa.DeviceSystem/HAL.cs

```diff
@@ -163,9 +163,20 @@ public static void Abort(string message)
 			hardwareAbstraction.Abort(message);
 		}
 
+		/// <summary>
+		/// Pause
+		/// </summary>
 		public static void Pause()
 		{
 			hardwareAbstraction.Pause();
 		}
+
+		/// <summary>
+		/// Halt the CPU.
+		/// </summary>
+		public static void HaltCPU()
```

Comment:
I don't see a reason for this to be part of the HAL interface given "acpi.Shutdown();"

Reply:
It's useful for some drivers, for example to wait until a device resets for example (if you send a request for it).

Reply:
Actually, this is calling x86 hlt which simply tells the CPU to go idle state (low energy state). Okay for now.